doc/man1/openssl-verification-options.pod

```diff
@@ -426,6 +535,133 @@ This option has no effect and is retained for backward compatibility only.
 
 =back
 
+=head2 Certificate Extensions
+
+Options like B<-purpose> lead to checking the certificate extensions,
+which determine what the target certificate and intermediate CA certificates
+can be used for.
+The checks for such restrictions are rather complex and include various
+hacks and workarounds to handle broken certificates and software.
+
+=head3 Basic Constraints
+
+The basicConstraints extension CA flag is used to determine whether the
+certificate can be used as a CA. If the CA flag is true then it is a CA,
+if the CA flag is false then it is not a CA. B<All> CAs should have the
+CA flag set to true.
+
+If the basicConstraints extension is absent then the certificate is
+considered to be a "possible CA" and other extensions are checked according
+to the intended use of the certificate. A warning is given in this case
+because the certificate should really not be regarded as a CA.
+However it is allowed to be a CA to work around some broken software.
```

Comment:
I had just moved that old text to here.
I fully agree that it is questionable if not outdated,
and I also wondered what warnings were meant here (while that text may originally referring to the `verify` app).

So I've replaced
> A warning is given in this case
because the certificate should really not be regarded as a CA.
However it is allowed to be a CA to work around some broken software.

by
> The treatment of certificates without basicConstraints as a CA
is presently supported, but may in the future be withdrawn.

and dropped
> If the certificate is a V1 certificate (and thus has no extensions) and
it is self-signed it is also assumed to be a CA but a warning is again given.
This is to work around the problem of Verisign roots
which are V1 self-signed certificates.

after extending a little further up:
> If the basicConstraints extension is absent then the certificate is
considered to be a "possible CA"
to 
> If the basicConstraints extension is absent, which includes the case that it is an X.509v1 certificate, then the certificate is
considered to be a "possible CA"